### PR TITLE
index field added as key to remove warnings

### LINKED ISSUE #5600: key Warnings in .map methods
**Describe the bug**
Currently, when visiting the Dashboard there is a warning named as 
"Each child in a list should have a unique key prop"
This Warning is coming in two methods ExportMenu and HospitalList

**To Reproduce**

1. Go to DashBoard Page
2. Open the Console 
3. See warning

**Expected behavior**
There Should be no warning while visiting the page 

**Screenshots**
Screenshot 2023-06-05 123731
Screenshot 2023-06-05 123723

### PR DESCRIPTION
**Pull request by mecskyverse**
### WHAT
In .map Methods of Files Export.tsx and HospitalList.tsx there is no key prop which is causing wanings in the console.

## Proposed Changes

- Fixes #5600 
- added Key Prop with index in Method of exportItems.map() in ExportList.tsx
- added Key Prop with index in Method of data.map in HospitalList.tsx

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [x] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [x] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [x] Completion of QA

### CODE REVIEW

**Comment by mecskyverse** [comment]
Using the index as a key prop was a bad Idea. I will be updating with a new field if possible or else I will delete the pull request.

**Comment by nihal467** [comment]
@khavinshankar @rithviknishad @Ashesh3 can anyone review it

File: src/Components/Common/Export.tsx [hunk 1]
```diff
@@ -47,8 +47,9 @@ export const ExportMenu = ({
         icon={<CareIcon className="care-l-import" />}
         className="bg-white hover:bg-primary-100 text-primary-500 enabled:border border-primary-500 tooltip"
       >
-        {exportItems.map((item) => (
+        {exportItems.map((item, index) => (
```
Comment (khavinshankar) [inline src/Components/Common/Export.tsx]: @mecskyverse prefer uuid for keys over index, can you check if there is item.id or any other unique identifier, if not use index

File: src/Components/Facility/HospitalList.tsx [hunk 2]
```diff
@@ -169,8 +169,8 @@ export const HospitalList = () => {
 
   let facilityList: any[] = [];
   if (data && data.length) {
-    facilityList = data.map((facility: any) => (
-      <FacilityCard facility={facility} userType={userType} />
+    facilityList = data.map((facility: any, index) => (
+      <FacilityCard key={index} facility={facility} userType={userType} />
```
Comment (khavinshankar) [inline src/Components/Facility/HospitalList.tsx]: @mecskyverse same here too

**Review by khavinshankar — CHANGES_REQUESTED** [review]

File: src/Components/Common/Export.tsx (hunk 1 above)
Comment (mecskyverse) [reply src/Components/Common/Export.tsx]: Ok sure I will check for it.

**Review by khavinshankar — APPROVED** [review]

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
@khavinshankar can you review and merge it

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**